src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -471,11 +472,16 @@ public void receiveCardUsed(AbstractCard c) {
 
     @Override
     public void receiveOnBattleStart(AbstractRoom abstractRoom) {
-        if (ReapAndSowSaveData.reapAndSowDamage != 0) {
-            AbstractDungeon.actionManager.addToBottom(
-                    new DamageAllEnemiesAction(AbstractDungeon.player, DamageInfo.createDamageMatrix(ReapAndSowSaveData.reapAndSowDamage, true),
-                            DamageInfo.DamageType.THORNS, AbstractGameAction.AttackEffect.SLASH_DIAGONAL));
-            ReapAndSowSaveData.reapAndSowDamage = 0;
```

Comment:
We should be able to delete `ReapAndSowSaveData` as well.

Reply:
Good catch.